examples/psyad/lfric/Makefile

```diff
@@ -47,3 +47,6 @@ run:
 clean:
 	cd tangent_linear; make clean
```

Comment:
I should have spotted this before but please can you do `${MAKE} -C tangent_linear` and similarly for all of the ones below.

Reply:
Done.